bouncer/bouncer.Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM public.ecr.aws/sam/build-python3.9:latest
+
+## Initial setup
+
+COPY ./bouncer ${LAMBDA_TASK_ROOT}/
+COPY ./models.py ${LAMBDA_TASK_ROOT}/
+COPY ./requirements-bouncer.txt ${LAMBDA_TASK_ROOT}/
```

Comment:
I think it would make more sense to move the `requirements.txt` right in the `bouncer` directory and name it `requirements.txt` not `requirements-bouncer.txt`